app/connector/UserDetailsConnector.scala

```diff
@@ -32,20 +30,22 @@ trait UserDetailsConnector {
   def getUserDetails(userDetailsUri: String)(implicit hc: HeaderCarrier) = http.GET[UserDetails](userDetailsUri)
 }
 
-object CredentialRole extends Enumeration {
-  type CredentialRole = Value
-  val User, Unknown = Value
+case class CredentialRole(value: String) extends AnyVal {
```

Comment:
@anshulbajpai case class extending `AnyVal` is to avoid object allocation at runtime.
@tamaslang `val` needs to be added to value